pyproject.toml

```diff
@@ -203,13 +202,6 @@ show_error_codes = true
 # strict_equality = true
 # # --strict end
 
-# black code formatting
-# https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
-[tool.black]
-line-length = 100
-preview = true
-required-version = "24.8.0"
```

Comment:
is there anything similar for ruff? This is kind of helpful for footgun stuff

Reply:
Unfortunately no. We've pinned the ruff version in the dev dependencies, but the ruff configuration doesn't confirm it. As ruff matures more, I think breaking changes will be less common.